floyd/cli/data.py

```diff
@@ -130,7 +132,7 @@ def print_data(data_sources):
 @click.argument('id', nargs=1)
 def clone(id):
     """
-    Download the code for the job to the current path
+    Download files from a dataset.
```

Comment:
`Download the files in a dataset`.  The word `from` sounds like you can download specific files in that dataset rather than the whole thing.